mifospay/src/main/java/org/mifospay/editprofile/ui/EditProfileScreen.kt

```diff
@@ -111,13 +112,14 @@ fun EditProfileScreen(
     updateSuccess: Boolean,
     uri: Uri?
 ) {
+    val showDialog = rememberSaveable { mutableStateOf(false) }
```

Comment:
rename it with `showDiscardChangesDialog`

Reply:
use by 
` val showDialog by rememberSaveable { mutableStateOf(false) }`

Reply:
Hello @therajanmaurya, I will rename the variable but the `MifosDialogBox` Composable requires a `mutableStateOf<Boolean>` as a parameter so can't use `by`.